### ISSUE TITLE
Snapshot testing with ava

### ISSUE THREAD

**Issue by jsbrain** [wallabyjs/public#2788]
### Issue description or question

I'm having a hard time trying to get snapshot testing to work with ava. I think this should be possible but somehow I just can't get it to work.

I always get:

```
No snapshot available - new snapshots are not created in CI environments
```

This is my config:

```js
// Ref: https://github.com/wallabyjs/ava-ts-sample/blob/master/wallaby.js
module.exports = function (wallaby) {
  // process.env.NODE_ENV = "test"; // <- not working
  // process.env.CI = "false"; // <- also not working

  return {
    files: [
      "src/**/*.ts",
      "src/**/*.json",
      "!src/**/*.spec.ts",
      { pattern: "src/**/snapshots/*.{md,snap}" },
    ],

    tests: ["src/**/*.spec.ts"],

    env: {
      type: "node",
    },

    compilers: {
      "**/*.[t|j]s?(x)": wallaby.compilers.typeScript({
        module: "commonjs",
        jsx: "React",
      }),
    },

    testFramework: "ava",

    filesWithNoCoverageCalculated: [".eslintrc.js"],

    maxConsoleMessagesPerTest: 150,
  };
};
```

My ava config in `package.json`:

```json
"ava": {
  "failFast": true,
  "timeout": "60s",
  "extensions": [
    "ts"
  ],
  "require": [
    "ts-node/register",
    "tsconfig-paths/register"
  ],
  "files": [
    "src/**/*.spec.ts",
    "!build/module/**"
  ]
},
```

Everything seems to be working with ava itself but simply Wallaby isn't picking up my snapshots.

I manually checked the instrumented directory and the snapshot files are being copied so it should work I guess but it doesn't ... what am I doing wrong here?

### Wallaby diagnostics report

```
{
  editorVersion: '1.59.1',
  pluginVersion: '1.0.311',
  editorType: 'VSCode',
  osVersion: 'darwin 20.5.0',
  nodeVersion: 'v16.4.2',
  coreVersion: '1.0.1136',
  checksum: 'YTdmYjg5NjYzMmU4ZjYwNTBhMjZlZDJmYzM2NDJiNTIsMTY0NDM2NDgwMDAwMCww',
  config: {
    files: [
      { pattern: 'src/**/*.ts', ignore: false, trigger: true, load: true, instrument: true, order: 1 },
      { pattern: 'src/**/*.json', ignore: false, trigger: true, load: true, instrument: true, order: 2 },
      { pattern: 'src/**/*.spec.ts', ignore: true, trigger: true, load: true, instrument: true },
      { pattern: 'src/**/snapshots/*.{md,snap}', ignore: false, trigger: true, load: true, instrument: true, order: 3 }
    ],
    tests: [ { pattern: 'src/**/*.spec.ts', ignore: false, trigger: true, load: true, test: true, order: 4 } ],
    env: { type: 'node', params: {}, runner: '/usr/local/bin/node', viewportSize: { width: 800, height: 600 }, options: { width: 800, height: 600 }, bundle: true },
    compilers: { '**/*.?(lit)coffee?(.md)': [Function (anonymous)] },
    testFramework: { version: 'ava@1.0.0', configurator: 'ava@1.0.0', reporter: 'ava@1.0.0', starter: 'ava@1.0.0', jsPreprocessor: [Function: jsPreprocessor] },
    filesWithNoCoverageCalculated: [ '.eslintrc.js' ],
    maxConsoleMessagesPerTest: 150,
    diagnostics: {},
    runAllTestsInAffectedTestFile: false,
    updateNoMoreThanOneSnapshotPerTestFileRun: false,
    addModifiedTestFileToExclusiveTestRun: true,
    preprocessors: {},
    autoConsoleLog: true,
    delays: { run: 0, edit: 100, update: 0 },
    workers: { initial: 0, regular: 0, recycle: false },
    teardown: undefined,
    hints: {
      ignoreCoverage: '__REGEXP /ignore coverage|istanbul ignore/',
      ignoreCoverageForFile: '__REGEXP /ignore file coverage/',
      commentAutoLog: '?',
      testFileSelection: { include: '__REGEXP /file\\.only/', exclude: '__REGEXP /file\\.skip/' }
    },
    automaticTestFileSelection: true,
    runSelectedTestsOnly: false,
    mapConsoleMessagesStackTrace: false,
    extensions: {},
    reportUnhandledPromises: true,
    slowTestThreshold: 75,
    lowCoverageThreshold: 80,
    loose: undefined,
    symlinkNodeModules: undefined,
    configCode: '// Ref: https://github.com/wallabyjs/ava-ts-sample/blob/master/wallaby.js\n' +
      'module.exports = function (wallaby) {\n' +
      '  // process.env.NODE_ENV = "test"; // <- not working\n' +
      '  // process.env.CI = "false"; // <- also not working\n' +
      '\n' +
      '  return {\n' +
      '    files: [\n' +
      '      "src/**/*.ts",\n' +
      '      "src/**/*.json",\n' +
      '      "!src/**/*.spec.ts",\n' +
      '      { pattern: "src/**/snapshots/*.{md,snap}" },\n' +
      '    ],\n' +
      '\n' +
      '    tests: ["src/**/*.spec.ts"],\n' +
      '\n' +
      '    env: {\n' +
      '      type: "node",\n' +
      '    },\n' +
      '\n' +
      '    compilers: {\n' +
      '      "**/*.[t|j]s?(x)": wallaby.compilers.typeScript({\n' +
      '        module: "commonjs",\n' +
      '        jsx: "React",\n' +
      '      }),\n' +
      '    },\n' +
      '\n' +
      '    testFramework: "ava",\n' +
      '\n' +
      '    filesWithNoCoverageCalculated: [".eslintrc.js"],\n' +
      '\n' +
      '    maxConsoleMessagesPerTest: 150,\n' +
      '  };\n' +
      '};\n'
  },
  packageJSON: {
    dependencies: {
      '@casl/ability': '^5.4.3',
      '@casl/prisma': '^1.0.0',
      '@jsbrain/utils': '<homeDir>/Code/@jsbrain/utils',
      ajv: '^8.6.2',
      'ajv-formats': '^2.1.0',
      'english-verbs-helper': '^2.0.1',
      'english-verbs-irregular': '^2.0.1',
      'json-pointer': '^0.6.1',
      'json-schema': '^0.3.0',
      'json-schema-traverse': '^1.0.0',
      'make-error': '^1.3.6'
    },
    devDependencies: {
      '@ava/typescript': '^2.0.0',
      '@istanbuljs/nyc-config-typescript': '^1.0.1',
      '@prisma/client': '^2.29.1',
      '@types/json-pointer': '^1.0.31',
      '@types/node': '^16.4.0',
      '@typescript-eslint/eslint-plugin': '^4.28.4',
      '@typescript-eslint/parser': '^4.28.4',
      alex: '^9.1.0',
      ava: '^4.0.0-alpha.2',
      benny: '^3.6.15',
      codecov: '^3.8.3',
      cspell: '^5.6.6',
      'cz-conventional-changelog': '^3.3.0',
      eslint: '^7.32.0',
      'eslint-config-prettier': '^8.3.0',
      'eslint-plugin-autofix': '^1.0.5',
      'eslint-plugin-eslint-comments': '^3.2.0',
      'eslint-plugin-functional': '^3.2.1',
      'eslint-plugin-import': '^2.23.4',
      'eslint-plugin-simple-import-sort': '^7.0.0',
      'gh-pages': '^3.2.3',
      i: '^0.3.6',
      klona: '^2.0.4',
      'npm-run-all': '^4.1.5',
      nyc: '^15.1.0',
      'open-cli': '^7.0.0',
      prettier: '^2.3.2',
      prisma: '^2.29.1',
      'prisma-json-schema-generator': '^1.4.0',
      'standard-version': '^9.3.1',
      'ts-node': '^10.2.1',
      'tsconfig-paths': '^3.10.1',
      'type-fest': '^2.0.0',
      typedoc: '^0.21.4',
      typescript: '4.4.1-rc'
    }
  },
  fs: { numberOfFiles: 21 },
  debug: [
    '2021-08-31T10:23:58.469Z project Wallaby Node version: v16.4.2\n',
    '2021-08-31T10:23:58.469Z project Wallaby config: <homeDir>/Code/@jsbrain/prisma-access-control/wallaby.js\n',
    '2021-08-31T10:23:58.501Z project File cache: <homeDir>/.vscode/extensions/wallabyjs.wallaby-vscode-1.0.311/projects/7a36461c6d9662b2\n',
    '2021-08-31T10:23:58.556Z uiService Listening port 51235\n',
    '2021-08-31T10:23:58.562Z project Config file change detected, invalidating local cache\n',
    '2021-08-31T10:23:58.670Z workers Parallelism for initial run: 18, for regular run: 9\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #0\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #1\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #2\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #3\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #4\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #5\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #6\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #7\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #8\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #9\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #10\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #11\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #12\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #13\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #14\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #15\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #16\n',
    '2021-08-31T10:23:58.670Z workers Starting run worker instance #17\n',
    '2021-08-31T10:23:58.671Z workers Web server is listening at 56513\n',
    '2021-08-31T10:23:58.671Z project File cache requires some updates, waiting required files from IDE\n',
    '2021-08-31T10:23:59.059Z workers Started run worker instance (delayed) #0\n',
    '2021-08-31T10:23:59.070Z workers Started run worker instance (delayed) #1\n',
    '2021-08-31T10:23:59.083Z workers Started run worker instance (delayed) #2\n',
    '2021-08-31T10:23:59.097Z workers Started run worker instance (delayed) #4\n',
    '2021-08-31T10:23:59.098Z workers Started run worker instance (delayed) #5\n',
    '2021-08-31T10:23:59.107Z workers Started run worker instance (delayed) #3\n',
    '2021-08-31T10:23:59.120Z workers Started run worker instance (delayed) #6\n',
    '2021-08-31T10:23:59.135Z workers Started run worker instance (delayed) #8\n',
    '2021-08-31T10:23:59.135Z workers Started run worker instance (delayed) #7\n',
    '2021-08-31T10:23:59.144Z workers Started run worker instance (delayed) #9\n',
    '2021-08-31T10:23:59.154Z workers Started run worker instance (delayed) #10\n',
    '2021-08-31T10:23:59.160Z workers Started run worker instance (delayed) #11\n',
    '2021-08-31T10:23:59.164Z workers Started run worker instance (delayed) #15\n',
    '2021-08-31T10:23:59.175Z workers Started run worker instance (delayed) #12\n',
    '2021-08-31T10:23:59.176Z workers Started run worker instance (delayed) #13\n',
    '2021-08-31T10:23:59.181Z workers Started run worker instance (delayed) #14\n',
    '2021-08-31T10:23:59.191Z workers Started run worker instance (delayed) #16\n',
    '2021-08-31T10:23:59.203Z workers Started run worker instance (delayed) #17\n',
    '2021-08-31T10:23:59.431Z project Stopping process pool\n',
    '2021-08-31T10:23:59.434Z project Running postprocessor\n',
    '2021-08-31T10:23:59.439Z postprocessor New TypeScript language service is required\n',
    '2021-08-31T10:24:01.156Z project Postprocessor execution finished\n',
    '2021-08-31T10:24:01.156Z project Test run started; run priority: 3\n',
    '2021-08-31T10:24:01.157Z project Running all tests\n',
    '2021-08-31T10:24:01.158Z workers Starting test run, priority: 3\n',
    '2021-08-31T10:24:01.158Z workers Distributing tests between 18 workers\n',
    '2021-08-31T10:24:01.158Z workers Running tests in parallel\n',
    '2021-08-31T10:24:01.158Z nodeRunner Starting sandbox [worker #0, session #16qec]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Starting sandbox [worker #1, session #72m64]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Starting sandbox [worker #2, session #fessf]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Starting sandbox [worker #3, session #rywxc]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Starting sandbox [worker #4, session #d2o8e]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Starting sandbox [worker #5, session #v83nl]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Preparing sandbox [worker #0, session #16qec]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Preparing sandbox [worker #1, session #72m64]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Preparing sandbox [worker #2, session #fessf]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Preparing sandbox [worker #3, session #rywxc]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Preparing sandbox [worker #4, session #d2o8e]\n',
    '2021-08-31T10:24:01.158Z nodeRunner Preparing sandbox [worker #5, session #v83nl]\n',
    '2021-08-31T10:24:01.159Z nodeRunner Prepared sandbox [worker #0, session #16qec]\n',
    '2021-08-31T10:24:01.159Z nodeRunner Prepared sandbox [worker #1, session #72m64]\n',
    '2021-08-31T10:24:01.159Z nodeRunner Prepared sandbox [worker #2, session #fessf]\n',
    '2021-08-31T10:24:01.159Z nodeRunner Prepared sandbox [worker #3, session #rywxc]\n',
    '2021-08-31T10:24:01.159Z nodeRunner Prepared sandbox [worker #4, session #d2o8e]\n',
    '2021-08-31T10:24:01.159Z nodeRunner Prepared sandbox [worker #5, session #v83nl]\n',
    '2021-08-31T10:24:01.159Z workers [worker #0, session #16qec] Running tests in sandbox\n',
    '2021-08-31T10:24:01.159Z workers [worker #1, session #72m64] Running tests in sandbox\n',
    '2021-08-31T10:24:01.159Z workers [worker #2, session #fessf] Running tests in sandbox\n',
    '2021-08-31T10:24:01.159Z workers [worker #3, session #rywxc] Running tests in sandbox\n',
    '2021-08-31T10:24:01.160Z workers [worker #4, session #d2o8e] Running tests in sandbox\n',
    '2021-08-31T10:24:01.160Z workers [worker #5, session #v83nl] Running tests in sandbox\n',
    '2021-08-31T10:24:01.412Z workers [72m64] Loaded unknown number of test(s)\n',
    '2021-08-31T10:24:01.413Z workers [72m64] Test executed: should throw custom error\n',
    '2021-08-31T10:24:01.413Z workers [72m64] Test executed: should return detailed error context\n',
    '2021-08-31T10:24:01.413Z workers [v83nl] Loaded unknown number of test(s)\n',
    '2021-08-31T10:24:01.414Z workers [v83nl] Test executed: should transform normal json schema to custom ability json schema\n',
    '2021-08-31T10:24:01.414Z workers [v83nl] Test executed: should fail for invalid schema in getFieldSubjectTypeMap\n',
    '2021-08-31T10:24:01.415Z workers [v83nl] Test executed: should get FieldSubjectTypeMap for abilitySchema\n',
    '2021-08-31T10:24:01.416Z workers [v83nl] Test executed: should get relevant fields of abilitySchema\n',
    '2021-08-31T10:24:01.416Z workers [v83nl] Test executed: should get past tense for words\n',
    '2021-08-31T10:24:01.420Z workers [72m64] Run 2 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:24:01.421Z workers [v83nl] Run 5 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:24:01.421Z workers [72m64] Sandbox is responsive, closing it\n',
    '2021-08-31T10:24:01.422Z workers [v83nl] Sandbox is responsive, closing it\n',
    '2021-08-31T10:24:01.440Z workers [16qec] Loaded unknown number of test(s)\n',
    '2021-08-31T10:24:01.440Z workers [16qec] Test executed: should handle `null` relations and relation lists (`[]`) + scalars\n',
    "2021-08-31T10:24:01.440Z workers [16qec] Test executed: should ignore list values that aren't a subject to support scalar lists\n",
    '2021-08-31T10:24:01.441Z workers [16qec] Test executed: should return detailed silent errors for the whole subject instance\n',
    '2021-08-31T10:24:01.441Z workers [16qec] Test executed: should return 0 errors after second run\n',
    '2021-08-31T10:24:01.441Z workers [16qec] Test executed: should return empty object if subject root is inaccessible\n',
    '2021-08-31T10:24:01.442Z workers [16qec] Test executed: should throw error on inaccessible root\n',
    '2021-08-31T10:24:01.442Z workers [16qec] Test executed: should test ability\n',
    '2021-08-31T10:24:01.442Z workers [16qec] Test executed: should transform query result\n',
    '2021-08-31T10:24:01.443Z workers [16qec] Test executed: should return fully tracked WeakMap\n',
    '2021-08-31T10:24:01.443Z workers [16qec] Test executed: should work with arrays as well\n',
    '2021-08-31T10:24:01.449Z workers [16qec] Run 10 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:24:01.450Z workers [16qec] Sandbox is responsive, closing it\n',
    '2021-08-31T10:24:01.461Z workers [d2o8e] Loaded unknown number of test(s)\n',
    '2021-08-31T10:24:01.461Z workers [d2o8e] Test executed: can parse big query\n',
    '2021-08-31T10:24:01.468Z workers [d2o8e] Run 1 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:24:01.469Z workers [d2o8e] Sandbox is responsive, closing it\n',
    '2021-08-31T10:24:01.508Z workers [fessf] Loaded unknown number of test(s)\n',
    '2021-08-31T10:24:01.508Z workers [fessf] Test executed: pac_traverse is really fast\n',
    '2021-08-31T10:24:01.508Z workers [fessf] Test executed: pac is not so fast\n',
    '2021-08-31T10:24:01.508Z workers [fessf] Test executed: new_traverse is fast!\n',
    '2021-08-31T10:24:01.513Z workers [rywxc] Loaded unknown number of test(s)\n',
    '2021-08-31T10:24:01.514Z workers [rywxc] Test executed: PacFilter works\n',
    '2021-08-31T10:24:01.514Z workers [rywxc] Test executed: PacFilter works like old filter\n',
    '2021-08-31T10:24:01.517Z workers [fessf] Run 3 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:24:01.517Z workers [fessf] Sandbox is responsive, closing it\n',
    '2021-08-31T10:24:01.518Z workers [rywxc] Run 2 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:24:01.519Z workers [rywxc] Sandbox is responsive, closing it\n',
    '2021-08-31T10:24:01.519Z workers Merging parallel test run results\n',
    '2021-08-31T10:24:01.522Z project Test run finished\n',
    '2021-08-31T10:24:01.523Z project Processed console.log entries\n',
    '2021-08-31T10:24:01.523Z project Processed loading sequences\n',
    '2021-08-31T10:24:01.523Z project Processed executed tests\n',
    '2021-08-31T10:24:01.526Z project Processed code coverage\n',
    '2021-08-31T10:24:01.565Z project Test run result processed and sent to IDE\n',
    '2021-08-31T10:24:07.786Z uiService UI client connected\n',
    '2021-08-31T10:24:07.787Z uiService Outgoing message ui:handshake\n',
    '2021-08-31T10:24:07.788Z uiService Incoming message ui:tests:resultsRequested\n',
    '2021-08-31T10:24:07.789Z uiService Outgoing message ui:tests:allResultsUpdated\n',
    '2021-08-31T10:24:07.789Z uiService Incoming message ui:start\n',
    '2021-08-31T10:24:07.789Z uiService Outgoing message ui:summary\n',
    '2021-08-31T10:24:07.790Z uiService Outgoing message ui:files\n',
    '2021-08-31T10:27:00.220Z fs File changed: src/lib/tests/query.spec.ts\n',
    '2021-08-31T10:27:00.222Z fs No changes detected for src/lib/tests/query.spec.ts\n',
    '2021-08-31T10:27:00.223Z extended-core File was not changed, but file markers may need to be synced\n',
    '2021-08-31T10:27:00.223Z uiService Outgoing message ui:summary\n',
    '2021-08-31T10:27:02.879Z project Requested to run all tests\n',
    '2021-08-31T10:27:02.880Z project Snapshots update test run\n',
    '2021-08-31T10:27:02.880Z uiService Outgoing message ui:busy\n',
    '2021-08-31T10:27:02.880Z project Running postprocessor\n',
    '2021-08-31T10:27:04.079Z project Postprocessor execution finished\n',
    '2021-08-31T10:27:04.079Z project Test run started; run priority: 3\n',
    '2021-08-31T10:27:04.079Z project Running all tests\n',
    '2021-08-31T10:27:04.080Z workers Starting test run, priority: 3\n',
    '2021-08-31T10:27:04.080Z workers Distributing tests between 18 workers\n',
    '2021-08-31T10:27:04.081Z workers Running tests in parallel\n',
    '2021-08-31T10:27:04.081Z nodeRunner Starting sandbox [worker #0, session #brmho]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Starting sandbox [worker #1, session #8jaym]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Starting sandbox [worker #2, session #rorh4]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Starting sandbox [worker #3, session #b28l9]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Starting sandbox [worker #4, session #w7eqm]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Starting sandbox [worker #5, session #lhlxu]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Preparing sandbox [worker #0, session #brmho]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Preparing sandbox [worker #1, session #8jaym]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Preparing sandbox [worker #2, session #rorh4]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Preparing sandbox [worker #3, session #b28l9]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Preparing sandbox [worker #4, session #w7eqm]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Preparing sandbox [worker #5, session #lhlxu]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Prepared sandbox [worker #0, session #brmho]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Prepared sandbox [worker #1, session #8jaym]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Prepared sandbox [worker #2, session #rorh4]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Prepared sandbox [worker #3, session #b28l9]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Prepared sandbox [worker #4, session #w7eqm]\n',
    '2021-08-31T10:27:04.081Z nodeRunner Prepared sandbox [worker #5, session #lhlxu]\n',
    '2021-08-31T10:27:04.081Z workers [worker #0, session #brmho] Running tests in sandbox\n',
    '2021-08-31T10:27:04.082Z workers [worker #1, session #8jaym] Running tests in sandbox\n',
    '2021-08-31T10:27:04.082Z workers [worker #2, session #rorh4] Running tests in sandbox\n',
    '2021-08-31T10:27:04.082Z workers [worker #3, session #b28l9] Running tests in sandbox\n',
    '2021-08-31T10:27:04.082Z workers [worker #4, session #w7eqm] Running tests in sandbox\n',
    '2021-08-31T10:27:04.083Z workers [worker #5, session #lhlxu] Running tests in sandbox\n',
    '2021-08-31T10:27:04.095Z workers [8jaym] Loaded unknown number of test(s)\n',
    '2021-08-31T10:27:04.097Z workers [8jaym] Test executed: should throw custom error\n',
    '2021-08-31T10:27:04.098Z workers [lhlxu] Loaded unknown number of test(s)\n',
    '2021-08-31T10:27:04.099Z workers [8jaym] Test executed: should return detailed error context\n',
    '2021-08-31T10:27:04.099Z workers [lhlxu] Test executed: should transform normal json schema to custom ability json schema\n',
    '2021-08-31T10:27:04.100Z workers [lhlxu] Test executed: should fail for invalid schema in getFieldSubjectTypeMap\n',
    '2021-08-31T10:27:04.100Z workers [lhlxu] Test executed: should get FieldSubjectTypeMap for abilitySchema\n',
    '2021-08-31T10:27:04.100Z workers [lhlxu] Test executed: should get relevant fields of abilitySchema\n',
    '2021-08-31T10:27:04.101Z workers [lhlxu] Test executed: should get past tense for words\n',
    '2021-08-31T10:27:04.105Z workers [8jaym] Run 2 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:27:04.105Z workers [lhlxu] Run 5 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:27:04.106Z workers [8jaym] Sandbox is responsive, closing it\n',
    '2021-08-31T10:27:04.106Z workers [lhlxu] Sandbox is responsive, closing it\n',
    '2021-08-31T10:27:04.119Z workers [brmho] Loaded unknown number of test(s)\n',
    '2021-08-31T10:27:04.120Z workers [brmho] Test executed: should handle `null` relations and relation lists (`[]`) + scalars\n',
    "2021-08-31T10:27:04.120Z workers [brmho] Test executed: should ignore list values that aren't a subject to support scalar lists\n",
    '2021-08-31T10:27:04.120Z workers [brmho] Test executed: should return detailed silent errors for the whole subject instance\n',
    '2021-08-31T10:27:04.121Z workers [brmho] Test executed: should return 0 errors after second run\n',
    '2021-08-31T10:27:04.121Z workers [brmho] Test executed: should return empty object if subject root is inaccessible\n',
    '2021-08-31T10:27:04.121Z workers [brmho] Test executed: should throw error on inaccessible root\n',
    '2021-08-31T10:27:04.121Z workers [brmho] Test executed: should test ability\n',
    '2021-08-31T10:27:04.122Z workers [brmho] Test executed: should transform query result\n',
    '2021-08-31T10:27:04.122Z workers [brmho] Test executed: should return fully tracked WeakMap\n',
    '2021-08-31T10:27:04.123Z workers [brmho] Test executed: should work with arrays as well\n',
    '2021-08-31T10:27:04.126Z workers [brmho] Run 10 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:27:04.128Z workers [brmho] Sandbox is responsive, closing it\n',
    '2021-08-31T10:27:04.138Z workers [w7eqm] Loaded unknown number of test(s)\n',
    '2021-08-31T10:27:04.139Z workers [w7eqm] Test executed: can parse big query\n',
    '2021-08-31T10:27:04.142Z workers [w7eqm] Run 1 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:27:04.143Z workers [w7eqm] Sandbox is responsive, closing it\n',
    '2021-08-31T10:27:04.156Z workers [rorh4] Loaded unknown number of test(s)\n',
    '2021-08-31T10:27:04.157Z workers [rorh4] Test executed: pac_traverse is really fast\n',
    '2021-08-31T10:27:04.157Z workers [rorh4] Test executed: pac is not so fast\n',
    '2021-08-31T10:27:04.157Z workers [rorh4] Test executed: new_traverse is fast!\n',
    '2021-08-31T10:27:04.158Z workers [b28l9] Loaded unknown number of test(s)\n',
    '2021-08-31T10:27:04.159Z workers [b28l9] Test executed: PacFilter works\n',
    '2021-08-31T10:27:04.159Z workers [b28l9] Test executed: PacFilter works like old filter\n',
    '2021-08-31T10:27:04.164Z workers [rorh4] Run 3 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:27:04.165Z workers [b28l9] Run 2 test(s), skipped 0 test(s)\n',
    '2021-08-31T10:27:04.165Z workers [rorh4] Sandbox is responsive, closing it\n',
    '2021-08-31T10:27:04.165Z workers [b28l9] Sandbox is responsive, closing it\n',
    '2021-08-31T10:27:04.166Z workers Merging parallel test run results\n',
    '2021-08-31T10:27:04.168Z project Test run finished\n',
    '2021-08-31T10:27:04.168Z project Processed console.log entries\n',
    '2021-08-31T10:27:04.168Z project Processed loading sequences\n',
    '2021-08-31T10:27:04.169Z project Processed executed tests\n',
    '2021-08-31T10:27:04.171Z project Processed code coverage\n',
    '2021-08-31T10:27:04.207Z project Test run result processed and sent to IDE\n',
    '2021-08-31T10:27:04.207Z uiService Outgoing message ui:summary\n',
    '2021-08-31T10:27:04.208Z uiService Outgoing message ui:tests:allResultsUpdated\n'
  ]
}
```

**Comment by smcenlly** [comment]
Thanks for reporting the problem. There was a breaking change since we added support for ava snapshots. This is fixed in Wallaby core `v1.0.1137`.

**Comment by jsbrain** [comment]
Works now, thanks a lot!

But I noticed, that when I use the Wallaby "Update Snapshots" shortcut, the snapshots are being saved as `js` but when I run ava they're being saved as `ts` resulting in duplicates if I want to use both ways of creating/updating them. The `ts` snapshots are being picked up and recognized by Wallaby but if I use the shortcut it will create the `js` duplicates.

Is there some way to configure the file extension being used for the snapshots?

**Comment by smcenlly** [comment]
Thanks for letting us know. We'll take a look at it early next week.

**Comment by smcenlly** [comment]
The incorrect snapshot file names for TypeScript files has been fixed in Wallaby core `v1.0.1139`. Thanks again for reporting the problem.

**Comment by jsbrain** [comment]
Awesome, thank you so much!

**Comment by jsbrain** [comment]
Well actually, after trying it just now it still creates `.js` files for me.
I'm on `v1.0.1140` now.

**Comment by smcenlly** [comment]
Could you please provide us with your diagnostics report? It may be related to specific package version(s) that you are using (we were using latest version of `ava`).

If it's not too hard to provide us with a sample repo that has the same problem, that would also help. In the mean time, we'll try some previous versions and see if we have the same problem.

**Comment by smcenlly** [comment]
We've reproduced the problem, appears to affect the latest beta but not earlier versions. We should have a fix in a few hours.

**Comment by smcenlly** [comment]
This is fixed for ava v4-alpha in Wallaby `v1.0.1141`.

**Comment by jsbrain** [comment]
Yes it's working now.
Thanks again for the amazing and quick support!
